src/web/pages/user-settings/helperFunctions.ts

```diff
@@ -13,6 +13,9 @@ import Languages from 'web/utils/Languages';
  * @returns {string|undefined} - The language name or null if not found
  */
 export const getLangNameByCode = (code: string): string | undefined => {
+  if (code === BROWSER_LANGUAGE) {
+    return Languages[BROWSER_LANGUAGE].name;
+  }
   const language = Languages[code];
   return isDefined(language) ? `${language.name}` : undefined;
```

Comment:
This should have worked already. Just replaced template literal with String now.
```suggestion
  const language = Languages[code];
  return isDefined(language) ? String(language.name) : undefined;
```